### PR TITLE
Updated PHP

### PR DESCRIPTION
**Pull request by GrahamCampbell**

### CODE REVIEW

File: ci_environment/phpbuild/attributes/default.rb [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
   :git => {
     :repository => "git://github.com/CHH/php-build.git",
-    :revision   => "42d218a33fc4b301c586ec631aa386e2cc2e0638"
+    :revision   => "edb42024864c7844523e3b5666e3088d116b5758"
   },
   :phpunit_plugin => {
     :git => {
-      :repository => "git://github.com/CHH/php-build-plugin-phpunit.git",
-      :revision   => "2ee717c0e783809458f3073258c63f42500d1298"
+      :repository => "git://github.com/php-build/phpunit-plugin.git",
```
Comment (GrahamCampbell) [inline ci_environment/phpbuild/attributes/default.rb]: We moved this repo here a while ago now.

**Comment by GrahamCampbell** [comment]
Grr.

**Comment by meatballhat** [comment]
@GrahamCampbell hey!  This should apply if you target `precise-stable` instead pretty please

**Comment by GrahamCampbell** [comment]
I did?

**Comment by GrahamCampbell** [comment]
Going to send a new PR since this is outdated now, again.

**Comment by meatballhat** [comment]
@GrahamCampbell Sorry for the changes.  And facepalm on me for not double checking first.  I'll get yours merged ASAF, although precise image updates are probably at least 2 weeks away yet.

**Comment by GrahamCampbell** [comment]
Sure. :)

**Comment by GrahamCampbell** [comment]
See https://github.com/travis-ci/travis-cookbooks/pull/528.

**Pull request closed without merging**